### ISSUE TITLE
runtime error on example config

### ISSUE THREAD

**Issue by duritong** [bakito/dns-checker#15]
Trying to follow the example config, always raises an error:

```
$ podman run  -p 2112:2112 -e TARGET=github.com  quay.io/bakito/dns-checker
time="2020-09-08T11:58:57Z" level=info msg=Starting interval=30s timeout=10s version=v3.0.0 workers=10
time="2020-09-08T11:58:57Z" level=info msg="Setup check" host=github.com
time="2020-09-08T11:58:57Z" level=info msg="Setup check" name=dns_checker_check
time="2020-09-08T11:58:57Z" level=info msg="Setup check" name=dns_checker_check
time="2020-09-08T11:58:57Z" level=info msg="Starting metrics" port=2112
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=1
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=2
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=3
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=4
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=5
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=6
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=7
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=8
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=9
time="2020-09-08T11:58:57Z" level=info msg="starting worker" worker=10
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x833000]

goroutine 55 [running]:
github.com/bakito/dns-checker/pkg/run.runCheck(0x9f0480, 0xc000294000, 0x6fc23ac00, 0xc000298000, 0xc000022027, 0xa, 0x0, 0x9ee840, 0xc000098570, 0x2)
	/build/pkg/run/execute.go:113 +0x120
github.com/bakito/dns-checker/pkg/run.(*worker).Start.func1(0xc00029c320)
	/build/pkg/run/pool.go:78 +0xca
created by github.com/bakito/dns-checker/pkg/run.(*worker).Start
	/build/pkg/run/pool.go:73 +0x3f
```

**Comment by duritong** [comment]
Same for the current master:

```
$ podman run  -p 2112:2112 -e TARGET=github.com  quay.io/bakito/dns-checker:master
Trying to pull quay.io/bakito/dns-checker:master...
Getting image source signatures
Copying blob f20f68829d13 done  
Copying blob 41ae95b593e0 done  
Copying blob 2340ba3dd6f5 done  
Copying blob b40ad573924e done  
Copying config 7bd355cf0e done  
Writing manifest to image destination
Storing signatures
time="2020-09-08T12:01:21Z" level=info msg=Starting interval=30s timeout=10s version=4b3ea76 workers=10
time="2020-09-08T12:01:21Z" level=info msg="Setup check" host=github.com
time="2020-09-08T12:01:21Z" level=info msg="Setup check" name=dns
time="2020-09-08T12:01:21Z" level=info msg="Setup check" name=probe-port
time="2020-09-08T12:01:21Z" level=info msg="Starting metrics" port=2112
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=1
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=2
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=3
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=4
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=5
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=6
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=7
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=8
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=9
time="2020-09-08T12:01:21Z" level=info msg="starting worker" worker=10
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x8333a0]

goroutine 38 [running]:
github.com/bakito/dns-checker/pkg/run.runCheck(0x9f04a0, 0xc00028c000, 0x6fc23ac00, 0xc000290000, 0xc000022027, 0xa, 0x0, 0x9ee860, 0xc0000aa420, 0x1)
	/build/pkg/run/execute.go:113 +0x120
github.com/bakito/dns-checker/pkg/run.(*worker).Start.func1(0xc00029e2c0)
	/build/pkg/run/pool.go:78 +0xca
created by github.com/bakito/dns-checker/pkg/run.(*worker).Start
	/build/pkg/run/pool.go:73 +0x3f
```

**Comment by duritong** [comment]
This got introduced with 95ad6fdbe6e0290101c875bdabfadcc472d68773

**Comment by duritong** [comment]
Fixed with v3.0.1
